### PR TITLE
3.12.13 owncloud

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@dschmidt the last version 3.12.13 is something totally different than the webdav-imports branch.

in other words - this is kind of a mess ...

### CODE REVIEW

**Comment by dschmidt** [comment]
It was rebased in between, this PR unfortunately doesn't make any sense afaict.

**Comment by dschmidt** [comment]
The PR was rebased in the meantime, so I'm afraid what you did here doesn't make any sense afaict.

Especially in this GitHub view you are not comparing the actual code but you see the diff applied if there were no conflicts (i.e. the accumulation of all individual commits which differ from the target branch). 

What you should do instead is:
```git diff v3.12.13-owncloud..webdav-imports```

Then you see a much more limited set of changes (which are actual upstream changes or adjustments to upstream changes).

In summary this PR basically tries to downgrade the PR branch, which we most probably don't want to do. If you want to do a followup release on the tested 3.12.13-owncloud version, please create a version/release branch for that.
